### PR TITLE
RENAMED Firmetix4Arduino to Firmetix-Library

### PR DESCRIPTION
**Pull request by Nilon123456789**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @Nilon123456789. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]

**Comment by MatteoPologruto** [comment]
Done:
```
2023/03/29 15:36:32 Loaded 5663 libraries from DB
Removing Firmetix4Arduino
Success!
```

**Pull request merged**
